tests/conftest.py

```diff
@@ -18,7 +21,22 @@ class User:
     email = attr.ib(type=str)
 
 
-@pytest.fixture(name='org')
+@pytest.fixture(autouse=True)
+def _verify_url(request, server_url, user, org):
```

Comment:
I'd prefer for us not to rely on private functionality from plugins. The leading underscore suggests, that this is not part of the public API of **pytest-base-url**.

We can replicate what it does here though

Reply:
Well this is replicated, I just changed what url it uses to ```server_url```.

Reply:
Do we use **pytest-base-url** at all then? I assume you were overwriting a private fixture here.

Reply:
No, I removed it, I just replicated this verification function.

Reply:
Ah, great! Thank you for clarifying this! 🙇